storage/namespace.go

```diff
@@ -522,7 +551,23 @@ func (n *dbNamespace) Flush(
 	return res
 }
 
-func (n *dbNamespace) NeedsFlush(blockStart time.Time) bool {
+func (n *dbNamespace) NeedsFlush(start time.Time, end time.Time) bool {
```

Comment:
Can we document the semantics w.r.t. the inclusiveness of the time boundaries? Do `start` and `end` need to be aligned on the block boundaries?